README.md

```diff
@@ -1,19 +1,31 @@
 [![Pursuit Logo](https://avatars1.githubusercontent.com/u/5825944?s=200&v=4)](https://pursuit.org)
 
-# [Assignment Title]
+# Pursuit-Core-Web-Promises-Fetch-Lab
 
 ## Getting Started
-* Fork this repo
-* Clone the forked repository
-* `cd` to the directory where you cloned it
-* `npm install` to install dependencies
-* `npm test` to open the cypress testing window
 
-> *Note*: Remember to `git add`, `git commit` and `git push` regularly
+- Fork this repo
+- Clone the forked repository
+- `cd` to the directory where you cloned it
+- `npm install` to install dependencies
+- `npm test` to open the cypress testing window
+
+> _Note_: Remember to `git add`, `git commit` and `git push` regularly
+
+## 1. MVP
+
+- A list of ten jokes that are randomly loaded when the page is refreshed, each with the `"card"` class
+- Clicking on a joke card should expand it and reveal the punchline
+- Clicking on it again should contract it and hide the punchline
+- A button with id `"reload"` that refreshes clears all the jokes on the screen and loads new random ones
+
+## 2. Bonus
```

Comment:
Only the MVP spec file should run on circle (not the bonus spec file).
You will need to add a `spec` property to the circle config yml. Example: https://github.com/joinpursuit/Pursuit-Core-Events-Lab/blob/master/.circleci/config.yml#L8